CONTRIBUTING.md

```diff
@@ -30,31 +30,79 @@ Install the recommended [plugins](./.vscode/extensions.json) for linter warnings
 
 `yarn build`
 
-### To open development server
+### To open development servers
 
 `yarn dev`
 
-### To run visual tests
+### To run all unit tests and visual tests
 
 `yarn test`
 
 _Before running this command, make sure Docker is running. See [Testing](#Testing) section below for more details._
 
-### To lint SCSS and fix autofixable errors
+### To lint and fix autofixable errors
 
 `yarn lint`
 
 ---
 
 ## Developing
 
-### Directory structure
+### Monorepo overview
 
-- **`src/` contains all the .scss files** with a folder for each component.
-  - After running build, the generated .css files are put in `lib/css/` folder (in project root).
-- **`backstop/tests/` contains the .html files** that uses those built styles.
-  - After running build, the minified .html files are put in `backstop/minified/` folder.
-- **`backstop/scenarios/` contains the .js files** where visual test scenarios are defined.
+We use [Turborepo](https://turborepo.org/) as our monorepo tool to improve the experience of [yarn workspaces](https://classic.yarnpkg.com/lang/en/docs/workspaces/). It allows running commands in parallel and caches build outputs.
+
+The root package.json includes a few commands that use `turbo run` to run the corresponding command in all workspaces.
+
+e.g. to build all workspaces together, run the following command from the root:
+
+```
+yarn build
+```
+
+#### Development environment
+
+To start the development server for all workspaces, run the following command from the root.
+```
+yarn dev
+```
+
+This will automatically build anything that's not already built, and run the `dev` script for every workspace in parallel, watching for file changes.
+
+By default, it will open the storybook and docs website in your browser. Any examples and playgrounds will need to be manually opened. The urls/ports are as follows (assuming they are available):
```

Comment:
HTML test pages also opened automatically. So mentioning them here. Also, not sure what was meant by "examples", so maybe we can remove it. 

```suggestion
By default, it will open the HTML test pages (`itwinui-css` demo), storybook (`itwinui-react` demo), docs website in your browser. Playgrounds will need to be manually opened. The urls/ports are as follows (assuming they are available):
```